frontend/src/metabase/home/components/EmbedHomepage/NeedMoreInfoCard.tsx

```diff
@@ -0,0 +1,37 @@
+import { jt, t } from "ttag";
+
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { Card, Text } from "metabase/ui";
+
+type NeedMoreInfoCardProps = {
+  embeddingDocsUrl: string;
+  analyticsDocsUrl: string;
+};
+
+export const NeedMoreInfoCard = ({
+  embeddingDocsUrl,
+  analyticsDocsUrl,
+}: NeedMoreInfoCardProps) => (
+  <Card px="xl">
+    <Text
+      lh={1.57}
+      color="text-dark"
+      fw="bold"
+    >{t`Need more information?`}</Text>
+    <Text lh={1.57} color="text-light" size="sm">
```

Comment:
how was 1.57 calculated?

Reply:
`desired line height in pixels from the design / font size`
before using it I checked and there was at least another place where it was set like that

Reply:
(Not a fan of the magic number, but we need to release this today)